xdsl/dialects/riscv.py

```diff
@@ -157,13 +164,87 @@ def print_parameter(self, printer: Printer) -> None:
 
 
 class RISCVOp(Operation, ABC):
-    pass
```

Comment:
Can you add a little docstring here that says what types of operations you support here?